coalib/bearlib/abstractions/ExternalBearWrap.py

```diff
@@ -6,9 +6,10 @@
 from coalib.bears.LocalBear import LocalBear
 from coala_utils.decorators import enforce_signature
 from coalib.misc.Shell import run_shell_command
-from coalib.results.Diff import Diff
+from coalib.results.Diff import Diff  # Ignore PyUnusedCodeBear
```

Comment:
I removed this, and the one below, and no build problems:
https://github.com/jayvdb/coala/commit/929bb200dcf45fc9a7fa4b56c5beb3d19777f46c
https://circleci.com/gh/jayvdb/coala/176

Technically, removing these unused imports is a "breaking change", but anyone importing Diff from here is being silly.